extlib/benz/read.c

```diff
@@ -640,7 +640,7 @@ read_nullable(pic_state *pic, xFILE *file, int c, struct reader_control *p)
 }
 
 static pic_value
-read(pic_state *pic, xFILE *file, int c, struct reader_control *p)
```

Comment:
I included `unistd.h` to use `posix_memalign`, which causes name conflict against `read` and write`.